scripts/convex.blocknative.ts

```diff
@@ -0,0 +1,64 @@
+import fetch from 'node-fetch'
```

Comment:
Could we change this to `micro-ftch` instead of `node-fetch`? You can see how it's used in this file: https://github.com/mds1/convex-shutdown-simulation/blob/89da1585e46cfb140b4de4813c54cdcb5e0f9c95/scripts/convex.tenderly.ts#L4

I like that it has fewer dependencies, plus want to make sure this is consistent with how the tenderly file sends HTTP requessts

Reply:
done, haven't seen this before - very clean!